### PR TITLE
Update the Page Lifecycle API guide

### PR DESCRIPTION
**Pull request by philipwalton**
This PR updates the Page Lifecycle API to reflect the current behavior in Chrome (and in anticipation of increased tab discarding that will roll out in 108).

### CODE REVIEW

File: site/en/blog/page-lifecycle-api/index.md [hunk 1]
```diff
@@ -76,26 +82,30 @@ state](#developer-recommendations-for-each-state) for the exceptions).
...
-  <figure>
-<a href="https://wd.imgix.net/image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png">
-    {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png", alt="Page Lifecycle API state and event flow.", width="800", height="541" %}
-    </a>
-  </figure>
+<figure>
+  <a href="https://wd.imgix.net/image/eqprBhZUGfb8WYnumQ9ljAxRrA72/1mN2saGWo57O6vIOdcGy.svg">
+    {% Img
+      src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/KCIeOsJ0lCWMthBSSBrn.svg",
+      alt="Page Lifecycle API state and event flow.",
```
Comment (malchata) [inline site/en/blog/page-lifecycle-api/index.md:89]: I'm a bit concerned this alt text isn't sufficient, but it _is_ a rather complicated diagram to depict fully in alt text.

File: site/en/blog/page-lifecycle-api/index.md [hunk 2]
```diff
@@ -76,26 +82,30 @@ state](#developer-recommendations-for-each-state) for the exceptions).
...
+</figure>
 
 ### States
 
 The following table explains each state in detail. It also lists the possible
 states that can come before and after as well as the events developers can
 use to observe changes.
 
-<table class="cyan">
+<table>
```
Comment (malchata) [inline site/en/blog/page-lifecycle-api/index.md:102]: This table should encapsulate header cells in a `<thead>` and body cells in a `<tbody>` for those reliant on AT.

File: site/en/blog/page-lifecycle-api/index.md [hunk 3]
```diff
@@ -76,26 +82,30 @@ state](#developer-recommendations-for-each-state) for the exceptions).
...
   <tr>
     <th>State</th>
     <th>Description</th>
   </tr>
   <tr>
-    <td><code id="state-active">Active</code></strong></td>
+    <td><strong id="state-active">Active</strong></td>
     <td>
       <p>A page is in the <i>active</i> state if it is visible and has
```
Comment (malchata) [inline site/en/blog/page-lifecycle-api/index.md:110]: Totally minor accessibility nit, but my understanding is that `<i>` or `<b>` elements don't convey the same semantics as `<em>` and `<strong>`. You can adjust these if necessary, but not a big deal if you do not.

File: site/en/blog/page-lifecycle-api/index.md [hunk 4]
```diff
@@ -281,14 +297,12 @@ that pertain to lifecycle and lists what states they may transition to and from.
...
-      <aside class="key-point">
-        {% Aside 'key-term' %}
-        A <code>blur</code> event does not necessarily signal a state change.
-        It only signals a state change if the page no longer has input focus
-        (i.e. the page did not just switch focus from one element to
-        another).
-        {% endAside %}
-      </aside>
+      <p>
+        <em><strong>Note:</strong> a <code>blur</code> event does not
```
Comment (malchata) [inline site/en/blog/page-lifecycle-api/index.md:301]: I see `<em>` and `<strong>` tags getting mixed in here. Probably best to keep these consistent.

File: site/en/blog/page-lifecycle-api/index.md [hunk 5]
```diff
@@ -862,93 +811,76 @@ detect the `pagehide` event and only use `unload` if the browser doesn't support
...
-
-  If you want to use the `beforeunload` event properly and in a
-  way that works cross-browser, the [PageLifecycle.js](https://github.com/GoogleChromeLabs/page-lifecycle)
-  library is our recommended solution.
-{% endAside %}
-
 ## FAQs
 
-### My page does important work when it's hidden, how can I stop it from being frozen or discarded?
+**Why isn't there a "loading" state?**
```
Comment (malchata) [inline site/en/blog/page-lifecycle-api/index.md:876]: Stylistic nit and you're free to reject it: but it seems there should be some introductory text in this section, and the questions should probably be another heading level deep (as they're already functioning visually as a header anyway).

**Review by malchata — APPROVED** [review]
Minor nits, otherwise LGTM.

File: site/en/blog/page-lifecycle-api/index.md (hunk 1 above)
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:89]: Added another sentence explaining that this graphic is a visualization of the content in the tables below (so it's clear that not being able to see this graphic doesn't prevent you from getting the information).

File: site/en/blog/page-lifecycle-api/index.md (hunk 2 above)
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:102]: Done.

File: site/en/blog/page-lifecycle-api/index.md (hunk 3 above)
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:110]: I'm not an expert on this, but I was using `<i>` to denote that these are technical terms, as suggested here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i

File: site/en/blog/page-lifecycle-api/index.md (hunk 5 above)
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:876]: I believe these are consistent throughout. The use of `<em>` is to visually separate these inline notes from the other  content.

File: site/en/blog/page-lifecycle-api/index.md (hunk 4 above)
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:301]: I believe these are consistent throughout. The use of `<em>` is to visually separate these inline notes from the other  content.

**Comment by malchata** [comment]
Gotcha. That makes sense!

File: site/en/blog/page-lifecycle-api/index.md [hunk 6]
```diff
@@ -973,20 +905,18 @@ frozen or discarded, see: [Heuristics for Freezing &amp; Discarding](https://doc
...
-### What is the Back-Forward Cache?
-  
-The Back-Forward Cache (or
-[bfcache](https://developer.mozilla.org/Firefox/Releases/1.5/Using_Firefox_1.5_caching)
-or [Page Cache](https://webkit.org/blog/427/webkit-page-cache-i-the-basics/))
-is a term used to describe a navigation optimization some browsers implement
-that makes using the back and forward buttons faster.
+<strong id="back-forward-cache">What is the back/forward cache?<strong>
+
+The [back/forward cache](https://web.dev/bfcache/) (or "bfcache" for short) is a
```
[the remark this answers is not in the document]
Comment (philipwalton) [reply site/en/blog/page-lifecycle-api/index.md:917]: I'll drop the use of bfcache.

**Pull request merged**